### PR TITLE
v7.2.0

### PR DESCRIPTION
**Pull request by DanailH**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,89 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.2.0
+
+_Apr 12, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Support reordering of items in `SimpleTreeView` (#12369) @flaviendelangle
```
Comment (flaviendelangle) [inline CHANGELOG.md:12]: Just realized that the PR title could be misunderstood, people could think that we support drag&drop on this component.
I would remove the highlight (even if it was a big task) and rename the PR "Fix behaviors when the item order changes"

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,89 @@
...
+
+#### `@mui/x-date-pickers-pro@7.2.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@7.2.0`.
+
+### Charts
+
+#### `@mui/x-charts@7.2.0`
+
+- [charts] Fix Barchart with empty dataset throwing an error (#12708) @JCQuintas
```
Comment (flaviendelangle) [inline CHANGELOG.md:59]:
```suggestion
- [charts] Fix BarChart with empty dataset throwing an error (#12708) @JCQuintas
```

or better

```suggestion
- [charts] Fix Bar chart with empty dataset throwing an error (#12708) @JCQuintas
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,89 @@
...
+- [DataGridPremium] Fix clipboard paste not working when cell loses focus (#12724) @cherniavskii
+
+### Date and Time Pickers
+
+#### `@mui/x-date-pickers@7.2.0`
+
+- [fields] Fix field editing after closing the picker (#12675) @LukasTy
+- [l10n] Improve French (fr-FR) locale (#12692) @FaroukBel
+- [l10n] Improve German (de-DE) locale (#12752) @Jens-Schoen
+- [l10n] Improve Swedish (sv-SE) locale (#12731) @pontusdacke
```
Comment (LukasTy) [inline CHANGELOG.md:48]:
```suggestion
- [l10n] Improve Swedish (sv-SE) locale (#12731) @pontusdacke
- [pickers] Fix desktop date time Pickers grid layout (#12748) @LukasTy
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,88 @@
...
+
+#### `@mui/x-date-pickers-pro@7.2.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@7.2.0`.
+
+### Charts
+
+#### `@mui/x-charts@7.2.0`
+
+- [charts] Fix Bar chart with empty dataset throwing an error (#12708) @JCQuintas
```
Comment (JCQuintas) [inline CHANGELOG.md:62]:
```suggestion
- [charts] Fix Bar chart with empty dataset throwing an error (#12708) @JCQuintas
- [charts] Fix `tickLabelInterval` not working on `YAxis` (#12746) @JCQuintas 
```

File: yarn.lock [hunk 5]
```diff
@@ -2097,6 +2097,28 @@
     prop-types "^15.8.1"
     react-is "^18.2.0"
 
+"@mui/x-license@7.1.1":
```
Comment (cherniavskii) [inline yarn.lock:2100]: Why was this added to the lockfile?

Comment (DanailH) [reply yarn.lock:2100]: good question - no idea. I can remove it tho.

**Review by cherniavskii — APPROVED** [review]

File: yarn.lock (hunk 5 above)
Comment (DanailH) [reply yarn.lock:2100]: done and done

File: yarn.lock [hunk 6]
```diff
@@ -2097,6 +2097,20 @@
     prop-types "^15.8.1"
     react-is "^18.2.0"
 
+"@mui/x-tree-view@7.1.1":
```
Comment (DanailH) [inline yarn.lock:2100]: I guess this should also be removed 
CC @flaviendelangle

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,93 @@
...
+
+- [DateTimeRangePicker] Fix desktop toolbar style (#12760) @LukasTy
+
+### Charts
+
+#### `@mui/x-charts@7.2.0`
+
+- [charts] Fix `tickLabelInterval` not working on YAxis (#12746) @JCQuintas
+- [charts] Fix Bar chart with empty dataset throwing an error (#12708) @JCQuintas
+- [charts] Fix `tickLabelInterval` not working on `YAxis` (#12746) @JCQuintas
```
Comment (JCQuintas) [inline CHANGELOG.md:63]:
```suggestion
- [charts] Fix Bar chart with empty dataset throwing an error (#12708) @JCQuintas
- [charts] Fix `tickLabelInterval` not working on `YAxis` (#12746) @JCQuintas 
```

File: yarn.lock (hunk 5 above)
Comment (DanailH) [reply yarn.lock]: hmm removing it fails the CI - it complains that there are unstaged changes :/

File: packages/x-tree-view-pro/package.json [hunk 8]
```diff
@@ -48,7 +48,7 @@
     "@mui/system": "^5.15.14",
     "@mui/utils": "^5.15.14",
     "@mui/x-license": "7.1.1",
```
Comment (LukasTy) [inline packages/x-tree-view-pro/package.json:50]:
```suggestion
    "@mui/x-license": "7.2.0",
```

**Review by LukasTy — APPROVED** [review]
🚀 💯
